src/dmd/dtoh.d

```diff
@@ -227,30 +227,43 @@ public:
     /// The generated header should contain comments for skipped declarations?
     const bool printIgnored;
 
-    /// Default linkage in the current scope (e.g. LINK.c inside `extern(C) { ... }`)
-    LINK linkage = LINK.d;
+    /// State specific to the current context which depends
+    /// on the currently visited node and it's parents
+    static struct Context
```

Comment:
Can the introduction of this be split out? As it looks to be a no-op change that doesn't affect codegen.

Reply:
Sure, I didn't do it initially because it would introduce a new type without actually using it ( aside from the `alias this` ).

Reply:
#12659